.github/workflows/release.yml

```diff
@@ -102,22 +102,22 @@ jobs:
     - name: Compress binary
       run: >
         gzip -9 --keep --force 
-        windows-binary/y-sweet-server.exe
-        linux-binary/y-sweet-server
-        macos-binary/y-sweet-server
+        windows-binary/y-sweet.exe
+        linux-binary/y-sweet
+        macos-binary/y-sweet
     - name: ls
       run: ls -la
     - name: rename and move
       run: >
-        mv windows-binary/y-sweet-server.exe.gz y-sweet-server-win-x64.exe.gz &&
-        mv linux-binary/y-sweet-server.gz y-sweet-server-linux-x64.gz &&
-        mv macos-binary/y-sweet-server.gz y-sweet-server-macos-x64.gz
+        mv windows-binary/y-sweet.exe.gz y-sweet-win-x64.exe.gz &&
+        mv linux-binary/y-sweet.gz y-sweet-linux-x64.gz &&
+        mv macos-binary/y-sweet.gz y-sweet-macos-x64.gz
     - name: Release
       uses: softprops/action-gh-release@v1
       with:
         name: ${{ github.event.inputs.version }}
         draft: true
         files: |
-            y-sweet-server-win-x64.exe.gz
-            y-sweet-server-linux-x64.gz
-            y-sweet-server-macos-x64.gz
```

Comment:
Do we need to update the npx wrapper? https://github.com/drifting-in-space/y-sweet/blob/main/js-pkg/server/src/get_binary.js#L16

Reply:
good catch.